style.css

```diff
@@ -1,99 +1,727 @@
+  border: 0.5px solid rgba(212, 212, 146, 0.336);
+  background-color: rgba(128, 128, 0, 0.185);
+  width: 38px;
+  height: 30px;
+  font-size: 18px;
+  padding: 5px;
 }
 
-.social-icons {
-  display: flex;
-  list-style: none;
-  justify-content: left;
-  justify-items: left;
+li.boot {
+  width: 5pc;
+}
+
+.cards_room {
+  margin-bottom: 50px;
+}
+
+.content_2 {
+  position: relative;
   width: 100%;
-  height: 16ch;
-  margin: 12px;
-  margin-bottom: 100px;
-  font-size: 26px;
 }
 
-.social-icons a {
-  color: #fff;
-  margin-right: 20px;
+.image {
+  display: block;
+  width: 100%;
+  height: auto;
 }
 
-.headline-page-h2 {
-  margin: 0 12px;
-  font-weight: bold;
-  line-height: 32px;
-  font-style: normal;
-  color: #fff5e1;
+.card {
+  position: absolute;
+  bottom: 0;
+  background: rgb(0, 0, 0);
+  background: rgba(0, 0, 0, 0.1); /* Black see-through */
+  color: #f1f1f1;
   width: 100%;
-  margin-top: 8px;
+  height: 100%;
+  transition: 0.5s ease;
+  opacity: 0;
+  font-size: 20px;
+  padding: 20px;
+  padding-top: 90px;
+  text-align: left;
 }
 
-.text-1 {
-  font-size: 16px;
+.content_2 .card {
+  opacity: 1;
+}
+
+/* .card {
+  padding-top: 90px;
+  text-align: left;
+} */
+
+.card ul {
+  list-style-type: none;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  padding-right: 20px;
+}
+
+.card li {
+  display: inline-block;
+  padding: 5px 5px 5px 5px;
+  background-color: rgba(252, 252, 252, 0.247);
+  height: 40px;
+  color: rgba(255, 255, 255, 0.637);
+}
+
+.cardyellow {
+  background-color: #ff6b00;
+  height: 40px;
+  text-align: center;
+  padding-top: 10px;
+  color: #f1f1f1;
+  font-size: 20px;
+}
+
+#content_one {
+  background-color: #262626;
+  background-image: url(images/Frame.png);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  background-size: 30% 20%;
+  background-position-x: 100%;
+  background-position-y: 15%;
+  color: #fff;
+}
+
+#yellow button {
+  width: 160px;
+  height: 50px;
+  background-color: rgba(255, 107, 0, 1);
+  color: white;
+  font-size: 17px;
+  font-style: normal;
+  font-weight: 700;
   line-height: 24px;
+  letter-spacing: 0.03em;
+  padding-top: 5px;
+  border: none;
+  margin-left: 0;
+  margin-top: 20px;
+}
+
+.content_4 p {
+  font-size: 25px;
+  text-align: center;
+  padding: 20px;
+  color: #091e42;
+  font-weight: 100;
   font-family: "Times New Roman", Times, serif;
-  font-weight: normal;
-  color: #fff;
-  margin: 12px 0;
-  margin-left: 12px;
-  width: 100%;
+}
+
+.line5 {
+  background-color: #f8f8f8be;
+  height: 3px;
+  margin-top: 130px;
+  margin-bottom: 30px;
+  border-radius: 2px;
+}
+
+.bottom_link {
+  padding-bottom: 50px;
+}
+
+.bottom_link h2 {
+  margin-left: 35%;
+}
+
+.bottom_link ul {
+  align-content: center;
+  padding-left: 50px;
+  margin-bottom: 40px;
+  margin-top: 40px;
+  margin-left: 30%;
+  list-style-type: none;
+}
+
+.bottom_link li {
+  margin-top: 15px;
+  font-size: 20px;
+}
+
+.content_3 {
+  padding: 40px;
+}
+
+form {
```

Comment:
- [ ] For mobile screen sizes, I believe a bug has been introduced while working on the desktop version. The contact form display has been altered for mobile screens.
![Screenshot from 2021-08-20 12-48-29](https://user-images.githubusercontent.com/65501535/130215349-9f3bf895-3d37-4f98-8b6b-e9a272a4351d.png)